enhancements/ingress/ingress-operator-operand-logging-level.md

```diff
@@ -153,8 +179,11 @@ who are running into Ingress issues in production.
 
 ### Implementation Details/Notes/Constraints [optional]
 
+#### Avoiding unnecessary pod rollouts
+
 Setting the logging level for IngressController pods should not cause the IngressController pods to rollout.
-This is expected behavior whenever an IngressController's pod template's environment variables are changed.
+Unfortunately, a new pod rollout is the expected behavior whenever an environment variable is changed in an
+IngressController's pod template.
 
 In place of an additional environment variable, openshift-router could watch for changes to a `ConfigMap`
```

Comment:
I don't quite get, are we proposing to watch for the configmap to avoid the rollouts or no? And if so, is there a way to do this dynamically for HAProxy (currently we are using the `global.log` command)?